src/http/os/http.cpp

```diff
@@ -73,7 +73,14 @@ class PlatformHTTP : public AbstractHTTP
             curl = curl_easy_init();
             if(curl) {
                 curl_easy_setopt(curl, CURLOPT_URL, ss.str().c_str()); // Set the URL that is about to receive our POST
-                curl_easy_setopt(curl, CURLOPT_POSTFIELDS, body); // Now specify the POST data ex: "username=baldninja"
+
+                /* make header with content-type */
+                struct curl_slist *header = NULL;
+                curl_slist_append(header, "Content-Type: application/json");
+
+                /* set header & body */
+                curl_easy_setopt(curl, CURLOPT_HTTPHEADER, header); // Set Header
+                curl_easy_setopt(curl, CURLOPT_POSTFIELDS, body); // Now specify the POST json data ex: "username=baldninja"
```

Comment:
typo in "boldninja", unless Rok is having hair issues, in which case my apologies for being insensitive.

Reply:
No typo 😂

https://dexplorer.ark.io:8443/api/v2/delegates/baldninja